Branch-SDK/Branch-SDK/BranchGetAppListRequest.h

```diff
@@ -1,16 +0,0 @@
-//
```

Comment:
we can get rid of BranchGetAppListRequest and BranchUpdateAppListRequest now? Just want to make sure this is not an accidental removal.